tools/ratbagd.py

```diff
@@ -303,6 +305,20 @@ def capabilities(self):
         return self._get_dbus_property("Capabilities")
 
     @GObject.property
+    def name(self):
+        """The name of the profile"""
+        return self._get_dbus_property("Name")
+
+    @name.setter
+    def name(self, name):
+        """Set the name of this profile.
+
+        @param name The new name, as string"""
+        if RatbagdProfile.CAP_WRITABLE_NAME in self.capabilities:
+            return self._set_dbus_property("Name", "s", name)
+        return RatbagErrorCode.RATBAG_ERROR_CAPABILITY
```

Comment:
I agree with @whot, but note also that returning values here doesn't actually work; the decorator makes it always return the property and not anything else. This has been a bug in the bindings for a long time (ever since I rewrote them, I think), but as it's nothing critical I haven't corrected it until now. Sorry!

Also, I prefer having the capability checks up in the client (e.g. Piper) and not in the bindings. Piper needs them anyway to show or hide certain configuration, and having them twice feels like a waste.